packages/app/cypress/e2e/specs_page.cy.ts

```diff
@@ -0,0 +1,353 @@
+import defaultMessages from '@packages/frontend-shared/src/locales/en-US.json'
+
+describe('Specs Page', () => {
+  describe('Component Workflows', {
+    viewportHeight: 768,
+    viewportWidth: 1024,
+  }, () => {
+    context('with storybook', () => {
+      beforeEach(() => {
+        cy.scaffoldProject('no-specs')
+        cy.openProject('no-specs')
+        cy.startAppServer('component')
+        cy.visitApp()
+
+        // With no specs present, the page renders two cards, one for creating from found components,
+        // another for creating from found stories.
+        cy.findAllByTestId('card').eq(0).as('ComponentCard')
+        .should('have.attr', 'tabindex', '0')
+        .within(() => {
+          cy.findByRole('heading', { level: 2, name: defaultMessages.createSpec.component.importFromComponent.header }).should('be.visible')
+          cy.contains(defaultMessages.createSpec.component.importFromComponent.description).should('be.visible')
+        })
+
+        cy.findAllByTestId('card').eq(1).as('StoryCard')
+        .should('have.attr', 'tabindex', '0')
+        .within(() => {
+          cy.findByRole('heading', { level: 2, name: defaultMessages.createSpec.component.importFromStory.header }).should('be.visible')
+          cy.contains(defaultMessages.createSpec.component.importFromStory.description).should('be.visible')
+        })
+      })
+
+      it('shows create first spec page with create from component and create from story options', () => {
+        cy.findByRole('heading', {
+          level: 1,
+          name: defaultMessages.createSpec.page.defaultPatternNoSpecs.title,
+        }).should('be.visible')
+
+        cy.findByTestId('create-spec-page-description').should('be.visible').and('contain', defaultMessages.createSpec.page.defaultPatternNoSpecs.component.description)
+
+        cy.get('@ComponentCard').should('be.visible')
+        cy.get('@StoryCard').should('be.visible')
+
+        cy.findByTestId('no-specs-message').should('be.visible').and('contain', defaultMessages.createSpec.noSpecsMessage)
+
+        cy.findByRole('link', { name: defaultMessages.createSpec.viewSpecPatternButton })
+        .should('be.visible')
+        .and('not.be.disabled')
+        .and('have.attr', 'href', '#/settings?section=project&setting=specPattern')
+      })
+
+      context('create from story', () => {
```

Comment:
Would it be worth throwing a test in to verify that clicking create from component correctly finds matches?

It does look like the stories are matched as well which makes it seem like we won't even need the create from story option...
<img width="647" alt="Screen Shot 2022-01-13 at 2 35 32 PM" src="https://user-images.githubusercontent.com/14099737/149405233-5e2eb9a1-eacb-4e30-868c-dc52a8affaf1.png">